2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,30 @@
+let list = document.getElementById("todo-list");
+
+while (list.hasChildNodes()) {
+  list.removeChild(list.firstChild);
+}
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
+  // let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
+    let li = document.createElement("li");
+    li.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+
+    let text = document.createTextNode(todo.task);
+
+    //line-through and delete buttons
+    let span = document.createElement("span");
+    span.className = "badge bg-primary rounded-pill";
+    span.innerHTML = `
+    <i class="fa fa-check" aria-hidden="true"></i>
+    <i class="fa fa-trash" aria-hidden="true"></i>
+    `;
```

Comment:
your list elements & icons are being rendered really nicely, but there seems to be some problems with the button click functionality- you can add event listeners to the elements (which listen for button clicks), for example:
```
  let trashIcon = document.createElement("i");
  trashIcon.className = "fa fa-trash";
  trashIcon.addEventListener("click", function () {
    list.removeChild(listItem);
  });
```